### PR TITLE
Convert RadBreakout to USE_PROB_PARAMS

### PR DESCRIPTION
**Pull request by maxpkatz**
## PR checklist

- [x] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

File: Exec/radiation_tests/RadBreakout/Prob_nd.F90 [hunk 1]
```diff
@@ -13,43 +13,11 @@ subroutine amrex_probinit(init, name, namlen, problo, probhi) bind(C, name="amre
   integer untin, i
```
Comment (zingale) [inline Exec/radiation_tests/RadBreakout/Prob_nd.F90:13]: untin can be removed

**Comment by zingale** [comment]
tests: http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2020-09-20/index.html

**Comment by zingale** [comment]
this is good by me once `untin` is removed.

File: Exec/radiation_tests/RadBreakout/Prob_nd.F90 (hunk 1 above)
Comment (maxpkatz) [reply Exec/radiation_tests/RadBreakout/Prob_nd.F90:13]: untin is also used for opening up the model file, so I kept it.

**Review by zingale — APPROVED** [review]

**Pull request merged**